turbinia/api/utils.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Turbinia API Server helper methods."""
+
+import logging
+import mmap
+import shutil
+import tempfile
+import os
+
+from fastapi import HTTPException
+from turbinia import config as turbinia_config
+
+log = logging.getLogger('turbinia:api_server:utils')
+
+
+def create_zip(request_id: str, task_id: str):
+  """Compress a Turbinia request or task's output directories
```

Comment:
Tiny nit:  According to the style guide, this should be on one line (and any additional text can go after a blank line).

Reply:
Fixed